### DISCUSSION TITLE
posts/tutorial-hrbust-2024/

### DISCUSSION THREAD

**Discussion by giscus** [F0rsyth1a/F0rsyth1a.github.io#2] [Announcements]
# posts/tutorial-hrbust-2024/

A. 罚时

签到

略
B. 数学题

思维，签到

输出n个9即可
ac code

1
2
3
4
5
6
7

#include<iostream>
int main()
{
    int n; std::cin >> n;
    for(int i = 1; i <= n; i ++) std::cout << 9;
    return 0;
}

C. 干杯！

字符串，签到

ac code

 1
 2
 3
 4
 5
 6
 7
 8
 9
10
11
12
13
14
15
16
17
18
19
20
21
22
23

#include<iostream>
#include<cstdlib>
#include<cstring>
#include<set>
using namespace std;

string s;
void solve()
{
    getline(cin, s);
    set<char> st;
    for(auto ch : s)
        if(isalpha(ch)) st.insert(ch);
    cout << st.size();
}
int main()
{
    ios::sync_with_stdio(false); cin.tie(0); cout.tie(0);
    int Case = 1;
    //cin >> Case;
    while(Case --) solve();
    return 0;
}

D. 向左看

单调栈

https://f0rsyth1a.github.io/posts/tutorial-hrbust-2024/

**Comment by 2258009564** [comment]
比我的命都难

**Comment by HowieHz** [comment]
%%%
